src/App.jsx

```diff
@@ -1,9 +1,20 @@
 import Footer from './components/Footer/Footer';
 import Header from './components/Header/Header';
+import { connect } from 'react-redux';
 import CurrentCityForecastView from './views/CurrentCityForecastView/CurrentCityForecastView';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import Geolocation from './services/GeolocationService';
+import ApiService from './services/ForecastApiService';
+import { changeLocation, setGeolocationCity } from './actions/locationsManagerActions';
+import Storage from './services/StorageConnectionService';
+import { CURRENT_LOCATION_STORAGE_CODE } from './utils/constants';
+
+function App(props) {
+  useEffect(async () => {
+    props.setGeolocationCity();
+  }, []);
```

Comment:
you'll need to have setGeolocationCity listed in the dependency array. Take a look here https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies